app/controllers/concerns/authentication_and_sso_concerns.rb

```diff
@@ -9,7 +9,7 @@ module AuthenticationAndSSOConcerns
 
   included do
     before_action :authenticate
-    before_action :set_api_cookie!
```

Comment:
We had this at first for the transition from auth headers to session cookie, so that users that were already signed in using auth headers could get a cookie set. We've fully transitioned by now, so this can be removed.